### ISSUE TITLE
Redirect uri error

### ISSUE THREAD

**Issue by GoNzCiD** [sladkovm/strava-oauth#1]
Hi, I've build the docker image and run container.

If I setup environment variable `APP_URL` to my service url, the method `/authorize` always return the error:
```log
{
  "message": "Bad Request",
  "errors": [
    {
      "resource": "Application",
      "field": "redirect_uri",
      "code": "invalid"
    }
  ]
}
```

If not it try to redirect to localhost, and It doesn't work, if I rewrite the url to my correct ip of docker host It shows correctly the expected json result.

**Comment by sladkovm** [comment]
Hi, 

the quick cross-check - does the domain of redirect url corresponds to the domain you set in the Strava application settings?

Otherwise, I've fixed some inconsistencies and updated the Readme. Please git pull and let me know whether new instructions helped to solve your problem.

**Comment by GoNzCiD** [comment]
Yes, I set the domain in strava app. I'll try and comment with the new changes.

**Comment by akpamaboris** [comment]
@GoNzCiD have you succeeded ?

**Comment by GoNzCiD** [comment]
@zeliwipin it was some time ago, and I can't remember... I was finding the OAuth for a concrete sample, but I don't remember if I use this or other implementations, or any adaptation... sorry :(

**Comment by akpamaboris** [comment]
@GoNzCiD : thank you, good news, my team finally succeeded implementing the Oauth system with Strava
